README.md

```diff
@@ -120,30 +130,29 @@ storyblokInit({
 
 `gatsby-source-storyblok` does three actions when you initialize it:
 
-- Provides a `useStoryblokState` custom hook in your app, that parses story content JSON into the object.
+- Provides a `StoryblokStory` that includes `useStoryblokState` that receives a story object.
```

Comment:
As I guess that `StoryblokStory` means using a component here, probably is it better to specify that or at least add `<StoryblokStory />` instead?

Reply:
Fixed👍 Thank you @Dawntraoz !